Makefile

```diff
@@ -0,0 +1,43 @@
+## While this script is running, you will see a file `running_silo.flag` created in the current directory.
```

Comment:
@Taepper do you want to create a PR with this makefile then I can approve it and it can be merged :-)